docs/components/Wrapper.jsx

```diff
@@ -3,6 +3,11 @@ import CozyTheme from '../../react/CozyTheme'
 import Paper from '../../react/Paper'
 import Button from '../../react/Button'
 import isTesting from '../../react/helpers/isTesting'
+import themes from '../../theme/themes'
+import Typography from '../../react/Typography'
+import Divider from '../../react/MuiCozyTheme/Divider'
+
+const isDevMode = process.env.BUILD_ENV === 'watch-styleguidist'
```

Comment:
There seems to be an existing function for that : https://github.com/cozy/cozy-ui/blob/master/scripts/build-utils.js#L1 . it is used during webpack build but maybe we can use it also in screenshots execution context?